src/Compute/Compute/Manual/VirtualMachineScaleSetCreateOrUpdateMethod.cs

```diff
@@ -681,7 +695,6 @@ async Task SimpleParameterSetExecuteCmdlet(IAsyncCmdlet asyncCmdlet)
             {
                 if (this.SecurityType?.ToLower() == ConstantValues.TrustedLaunchSecurityType || this.SecurityType?.ToLower() == ConstantValues.ConfidentialVMSecurityType)
                 {
-                    this.SecurityType = this.SecurityType;
```

Comment:
why is this line being removed?  Do we want to assign if TrustLaunch or ConfidentialVM security type?

Reply:
It's logically unnecessary.